samples/dashboards/demo/cloud-monitoring/demo.html

```diff
@@ -0,0 +1,26 @@
+<script src="https://code.highcharts.com/dashboards/dashboards.js"></script>
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+<script src="https://code.highcharts.com/highcharts-more.js"></script>
+<script src="https://code.highcharts.com/modules/solid-gauge.js"></script>
+<script src="https://code.highcharts.com/dashboards/datagrid.js"></script>
+<script src="https://code.highcharts.com/dashboards/modules/dashboards-plugin.js"></script>
+
+<div class="cloud-monitoring-dashboard-container">
+  <div class="cloud-monitoring-data-controls">
+    <label for="enablePolling">
+      <input type="checkbox" id="enablePolling" name="enablePolling" />
```

Comment:
The polling feature works so well now, and adds a wow factor to this demo. It should be enabled by default. It's too good to hide behind an opt-in. At the same time, we could consider moving this checkbox to a less prominent place.